src/desktop/lib/global_react_modules.js

```diff
@@ -1,6 +1,16 @@
 import React from 'react'
 import { Artwork as _Artwork } from '@artsy/reaction/dist/Components/Artwork'
 import { ArtworkGrid as _ArtworkGrid } from '@artsy/reaction/dist/Components/ArtworkGrid'
+import { Tooltip as ReactionTooltip } from '@artsy/reaction/dist/Components/Tooltip'
+import { Help as ReactionHelpIcon } from '@artsy/reaction/dist/Assets/Icons/Help'
```

Comment:
This file currently contains some WIP stuff around the grid, hence the reason why I have a few components imported and then wrapped as an additional export -- since you're not making any modifications before data is being passed to reaction, can do 

```javascript
export { Tooltip } from '@artsy/reaction/dist/Components/Tooltip'
export { Help } from '@artsy/reaction/dist/Assets/Icons/Help'
```